packages/core-scroll/core-scroll.js

```diff
@@ -77,10 +77,16 @@ export default class CoreScroll extends HTMLElement {
     window.addEventListener('load', this) // Update state when we are sure all CSS is loaded
     document.addEventListener('click', this)
     setTimeout(() => this.handleEvent()) // Initialize buttons after children is parsed
```

Comment:
Would it make more sense to move the setup for `this._childListObserver` above the invocation of `this.handleEvent`, in order to keep the `setTimeout` as the last point in `connectedCallback` to match with the intention behind `setTimeout`?

Reply:
absolutely!